ajax/libs/arrive/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "arrive",
+  "filename": "minified/arrive.min.js",
+  "version": "2.3.1",
+  "description": "arrive.js provides events to watch for DOM elements creation and removal. It makes use of Mutation Observers internally.",
+  "repository": {
+    "type": "git",
+    "url": "git://git@github.com/uzairfarooq/arrive.git"
```

Comment:
Just a reminder, `git` protocol on GitHub with `git`(@) user info here won't work, if it comes from upstream, you can send a pull request to help them fix it.